PythonVisualizations/BinaryTreeBase.py

```diff
@@ -0,0 +1,515 @@
+from tkinter import *
+import time, random
+from enum import Enum
+
+try:
+    from drawable import *
+    from VisualizationApp import *
+except ModuleNotFoundError:
+    from .drawable import *
+    from .VisualizationApp import *
+
+class Child(Enum):
+    LEFT = 0
+    RIGHT = 1
+
+class Node(object):
+   def __init__(self, drawable, coords, tag):
+      self.drawable = drawable
+      self.coords = coords
+      self.tag = tag
+
+   def getKey(self):
+      return self.drawable.val
+
+   def setKey(self, k):
+      self.drawable.val = k
+
```

Comment:
There's a bit of asymmetry between defining get and set methods for the key, but not for the other attributes of the `Node`.  Since the key is inside the `drawable` record, it makes some sense, but it might be "cleaner" if either:
 * there were get/set pairs for all the attributes 
 * the `Node` class had individual attributes like those of the `drawable` (i.e. val, color, display_shape, display_val)

This isn't something that has to change.  I just want to point out the distinction.